### ISSUE TITLE
Why is the accuracy in the training set and test set both about 10%?

### ISSUE THREAD

**Issue by Cro-allen** [stefanonardo/pytorch-esn#18]
I have not changed any code of mnist in the examples. Why is the accuracy in the training set and test set both about 10%?

**Comment by stefanonardo** [comment]
@Cro-allen the model is not "trained". I added the code to show an example of working with bigger datasets.

**Comment by Cro-allen** [comment]
Thank you for your reply, and where can I find classification examples for the bigger datasets?

**Comment by stefanonardo** [comment]
Not in this repository I'm afraid. You can take a look at my thesis, I trained ESNs on bigger datasets. You find the link in the README.

**Comment by Shadab442** [comment]
If I want to improve the accuracy (say around 95%) for classification on the MNIST dataset, do I need to tune the hyperparameters? Or do I need to change anything in the implementation? Can you give some insights about what I may need to improve?
